tests/callbacks/CMakeLists.txt

```
@@ -1 +1,8 @@
+# List the header files and add them to the global property of files for the documentation.
+set(TESTS_CALLBACKS_HEADER_LIST
+    ${CMAKE_CURRENT_SOURCE_DIR}/TestWorkload.hpp
+)
+set_property(GLOBAL APPEND PROPERTY KokkosUtils_FILES_FOR_DOC ${TESTS_CALLBACKS_HEADER_LIST})
```

This is more than enough:
```suggestion
set_property(GLOBAL APPEND PROPERTY KokkosUtils_FILES_FOR_DOC ${CMAKE_CURRENT_SOURCE_DIR}/TestWorkload.hpp)
```

---

OK. Done!